feature/settings/src/main/java/com/google/jetpackcamera/settings/ui/SettingsComponents.kt

```diff
@@ -240,6 +242,61 @@ fun CaptureModeSetting(currentCaptureMode: CaptureMode, setCaptureMode: (Capture
     )
 }
 
+@Composable
+fun TargetFpsSetting(
+    modifier: Modifier = Modifier,
+    currentTargetFps: TargetFrameRate,
+    setTargetFps: (TargetFrameRate) -> Unit
+) {
+    BasicPopupSetting(
```

Comment:
We need to check against the supported FPS ranges in `CameraInfo.getSupportedFrameRateRanges()` to check which options we should show here. We don't need to show all of the ranges, but at least need to match them up to 15, 30, 60.

Reply:
right now i dont think the default settings are connected to the camera. I'll need to figure out how to send that information from the camera to settings, since its only source of information currently comes from the datastore.

I could have the cameraxCameraUseCase populate the datastore with information regarding device available fps at init, similar to the information for front or rear camera availability.

Reply:
That's a good idea. We should populate the datastore from CameraUseCase somehow. We should then check if the 3 options of 15, 30 and 60 are supported, and hide them as an option in the UI if they're not supported.